client/src/request.rs

```diff
@@ -267,4 +286,78 @@ impl SuccinctClient {
             Err(Error::msg("Failed to submit request to Succinct X API."))
         }
     }
+
+    pub async fn submit_request(
+        &self,
+        chain_id: u32,
+        to: Address,
+        calldata: Bytes,
+        function_id: B256,
+        input: Bytes,
+    ) -> Result<String> {
+        if self.local_mode() {
+            return self.submit_local_request(
+                chain_id,
+                to,
+                calldata.clone(),
+                function_id,
+                input.clone(),
+            );
+        }
+
+        self.submit_platform_request(chain_id, to, calldata, function_id, input)
+            .await
+    }
+
+    pub async fn submit_proof(&self, ethereum_rpc_url: String, request_id: String) -> Result<()> {
```

Comment:
Should we have another flag in the client that is like `RELAY=true` or something like that where if you're on local mode and relay=true, then after the proof generates, it automatically calls `submit_proof` or something like this? 

Btw I think that the `ethereum_rpc_url` should be read from the .env based on what chain_id is provided in the original request and panic if it is not found. Also for the gateway address, we should provide an option for people to specify that in their .env and fallback to our defaults if it exists since there might be chains where people deploy their own gateways.

Reply:
Thought about this, I don't think chain id should be read from `.env`, people might have their `.env` formatted differently, and forcing people to have things like `RPC_URL` in their `.env` in our format doesn't seem correct.

I'll add the `RELAY_MODE`.